packages/components/src/List/Toolbar/Filter/Filter.component.js

```diff
@@ -1,68 +1,161 @@
 import React, { PropTypes } from 'react';
+import classNames from 'classnames';
 import DebounceInput from 'react-debounce-input';
 import FormControl from 'react-bootstrap/lib/FormControl';
-import Button from 'react-bootstrap/lib/Button';
+import { Action } from '../../../Actions';
 import Icon from '../../../Icon';
+import theme from './Filter.scss';
+
+const ESC_KEY = 27;
+const ENTER_KEY = 13;
```

Comment:
I'm starting to see those kind of key handling constants more and more. 
Do you think we should extract them into a shared library?